process/smartContract/process.go

```diff
@@ -368,6 +368,10 @@ func (sc *scProcessor) executeSmartContractCall(
 		return userErrorVmOutput, sc.ProcessIfError(acntSnd, txHash, tx, err.Error(), []byte(returnMessage), snapshot, vmInput.GasLocked)
 	}
 
+	if sc.enableEpochsHandler.IsMaxBlockchainHookCountersFlagEnabled() {
```

Comment:
I think we can call reset without the flag usage. Flags are not taken into account anyway in the blockchain hook before the activation epoch. Valid on all calls.

Reply:
done